AzureIndexer/Stratis.Bitcoin.Features.AzureIndexer/AzureIndexer.cs

```diff
@@ -24,13 +25,18 @@ public enum IndexerCheckpoints
 
     public class AzureIndexer
     {
-        public static AzureIndexer CreateIndexer(IConfiguration config)
+        public static AzureIndexer CreateIndexer(IConfiguration config, ILoggerFactory loggerFactory)
         {
-            var indexerConfig = new IndexerConfiguration(config);
+            var indexerConfig = new IndexerConfiguration(config, loggerFactory);
             return indexerConfig.CreateIndexer();
         }
 
         private readonly IndexerConfiguration _Configuration;
```

Comment:
Underscore  isn't valid coding style.

Reply:
Totally agreed. Didn't touch it because if I would I'd rewrite half of the files in the AzureIndexer.
Hope the one who will work on the azure indexer will fix the code style.